Updates automated workflow with initial setup information (#566)
ansible-community/ansible-build-data: docs/automated-process.md

    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
    specify.

## Setup for the release

> ### Credentials

Reviewer: The items in this section are mostly things you cannot do yourself, but you need others to give to you (membership in GH repo and GH group). Maybe this should be explicitly stated, before someone asks "how do I do this?"?

Maybe:
```suggestion
### Credentials

Note that most of the following items cannot be done yourself, but need someone from the Ansible community team to assign them to you. You need to earn some trust first before this will happen.
```

> - Membership to [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.

Reviewer: Since the other two points are formulated as actions, this one should probably be one as well:
```suggestion
- Become a member of the [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.
```

- Become member of the [Ansible Release Management working group](https://github.com/orgs/ansible-community/teams/release-management-wg/).

> - Join [Release Management Working Group in [Ansible Forum](https://forum.ansible.com/g/release-managers) and [Matrix](#release-management:ansible.com).

Reviewer: For the release announcements, you need to be member of more forum groups. Should these also be mentioned here?

### Read about the following

- [Trusted Publisher in PyPI](https://docs.pypi.org/trusted-publishers/).
- [Examine the GitHub Actions release workflow](https://github.com/ansible-community/ansible-build-data/blob/main/.github/workflows/ansible-release.yml).
- [Talk on Using Trusted Publishing to Ansible release](https://docs.pypi.org/trusted-publishers/)
- Ask and show intention to be the Release Manager in the release-management working group.
- Shadow the release manager for 2 releases.
- [Release Roadmap](https://docs.ansible.com/ansible/devel/roadmap)

### Process summary

-  Communicate with the Community about the start and the progress on the [#release-management:ansible.com Matrix channel](https://matrix.to/#/#release-management:ansible.com).
-  Follow the release workflow as mentioned below.

> -  Release Announcement at the forum.

Reviewer: The last point is a subset of the middle point: step 6 of the instructions is about announcing the release on Matrix and on the forum.

Author: Updated.

    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
    specify.

## Setup for the release

### Credentials

Note that most of the following items cannot be done yourself, but need someone from the Ansible community team to assign them to you. You need to earn some trust first before this will happen.

- Become a member of the [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.
- Become member of the [Ansible Release Management working group](https://github.com/orgs/ansible-community/teams/release-management-wg/).

> - Join [Release Management Working Group in [Ansible Forum](https://forum.ansible.com/g/release-managers) and [Matrix](#release-management:ansible.com). For making the announcements relating to releases one needs to be a member of the Release Management Working Group in Forum.

Author: Updated with the following:

```
- Become member of the [Ansible Release Management working group](https://github.com/orgs/ansible-community/teams/release-management-wg/). For making the announcements relating to releases, please open your Matrix client and send the message to:
  - #release-management:ansible.com
  - #community:ansible.com
  - #packaging:ansible.com
  - #social:ansible.com (mention @newsbot)
  ```

    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
    specify.

## Setup for the release

### Credentials

> - Access to https://github.com/ansible-community/ansible-build-data repo

Reviewer 2: Specifically, they need to be a member of https://github.com/orgs/ansible-community/teams/release-management-wg/ (@ansible-community/release-management-wg). That gives the necessary permissions to this repo

> - Access to https://github.com/ansible/ansible repo

Reviewer 2: What do you mean by access? Nothing needs to be committed to ansible/ansible as part of the community release process.

> - Access to Ansible Release Management working group (in Github).

Reviewer 2: I think "membership" is probably a better word. Also, Release Management WG members should join the forum group.

- Join Release Management working group and Community Working Group.

> - Be a part of the maintainer group in PyPI.

Reviewer 2: I don't think PyPI access is strictly required anymore, unless something goes wrong and the process needs to be completed manually.

Reviewer: I think the idea is to explicitly **not** have access to PyPI for most people.

> - Access to test PyPI.

Reviewer 2: The current process doesn't rely on test PyPI. Other than @anweshadas, I don't think any of the current release manager have access to the Ansible project there.

Author: Removed the PyPI section.

> - Become member of the [Ansible Release Management working group](https://github.com/orgs/ansible-community/teams/release-management-wg/). For making the announcements relating to releases, please open your Matrix client and send the message to:

Reviewer 2: ```suggestion
- Become member of the Ansible Release Management working group on Github.
- For making the announcements relating to releases, please open your Matrix client and send the message to:
```

>   - #release-management:ansible.com
>   - #community:ansible.com
>   - #packaging:ansible.com
>   - #social:ansible.com (mention @newsbot)

Reviewer 2: ```suggestion
    - [`#release-management:ansible.com`](https://matrix.to/#/#release-management:ansible.com)
    - [`#community:ansible.com`](https://matrix.to/#/#community:ansible.com)
    - [`#packaging:ansible.com`](https://matrix.to/#/#packaging:ansible.com)
    - [`#social:ansible.com`](https://matrix.to/#/#social:ansible.com) (mention `@newsbot`)
```

### Read about the following

- [Trusted Publisher in PyPI](https://docs.pypi.org/trusted-publishers/).
- [Examine the GitHub Actions release workflow](https://github.com/ansible-community/ansible-build-data/blob/main/.github/workflows/ansible-release.yml).
- [Talk on Using Trusted Publishing to Ansible release](https://docs.pypi.org/trusted-publishers/)
- Ask and show intention to be the Release Manager in the release-management working group.
- Shadow the release manager for 2 releases.
- [Release Roadmap](https://docs.ansible.com/ansible/devel/roadmap)

### Build Checklist

> -  Communicate with the Community about the start and the progress on the #release-management:ansible.com Matrix channel.

Reviewer 2: Maybe includes an https://matrix.to link to join the channel?

> -  Fork `ansible-community/ansible-build-data` repo.

Reviewer 2: This is currently handled by the release workflow.

-  Fork `ansible/ansible-documentation` repo.
-  Follow the workflow from the ansible-build-data repo.

> -  Release Announcement at the forum.

Reviewer 2: Is this checklist redundant given the information is all below? Maybe the heading should be renamed `Process summary` and then there should be a sentence before the checklist that this summarizes the process outlined below?

Author: Thank you @felixfontein, updated.

    `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
    specify.

## Setup for the release

### Credentials

Note that most of the following items cannot be done by yourself, but need someone from the Ansible community team to assign them to you. You need to earn some trust first before this will happen.

- Become a member of the [ansible-build-data](https://github.com/ansible-community/ansible-build-data) repo.
- Become member of the Ansible Release Management working group on Github.

> - For making the announcements relating to releases, please open your Matrix client and send the message to:

Reviewer: @gotmax23 since this is not about sending announcements, but about which rooms to join (so you can later send announcements), I think this sentence should be changed:
```suggestion
- For making the announcements relating to releases, please join the following Matrix rooms:
```

Author: I see and agree with your point @felixfontein , I have updated  as recommended by you.
